public/src/js/widgets/fetch-latest-packages.js

```diff
@@ -0,0 +1,37 @@
+import Extension from 'models/extension';
+import {request} from 'modules/authed-ajax';
+import CONST from 'constants/defaults';
+
+export default class extends Extension {
+    constructor(baseModel) {
+        super(baseModel);
+        this.baseModel = baseModel;
+        this.fetchPackages();
+        this.pollPackages();
+        this.pollingId;
+    }
+
+    fetchPackages() {
+        var that = this;
+        return request({
+            url: CONST.apiBase + '/story-packages/latest'
```

Comment:
since the extension has access to the base model, we should send the request parameter `isHidden`, which is `true` if the base model priority is `training`